dali/test/test_tensors.h

```diff
@@ -66,7 +66,7 @@ class TestTensorList {
     if (!gpumem_) {
       auto size = shape_.num_elements() * sizeof(T);
       char *ptr = nullptr;
-      cudaMalloc(reinterpret_cast<void**>(&ptr), size);
+      CUDA_CALL(cudaMalloc(reinterpret_cast<void**>(&ptr), size));
```

Comment:
The "memory problem" was that allocation of such a big chunk of memory simply fails but it wasn't checked by TestTensorList